Block/Paypal/Shortcut/Button.php

```diff
@@ -0,0 +1,22 @@
+<?php
+/**
+ *
+ * Adyen ExpressCheckout Module
+ *
+ * Copyright (c) 2024 Adyen N.V.
+ * This file is open source and available under the MIT license.
+ * See the LICENSE file for more info.
+ *
+ * Author: Adyen <magento@adyen.com>
+ */
+declare(strict_types=1);
+
+namespace Adyen\ExpressCheckout\Block\Paypal\Shortcut;
+
+use Adyen\ExpressCheckout\Block\Buttons\AbstractButton;
+use Magento\Catalog\Block\ShortcutInterface;
+
+class Button extends AbstractButton implements ShortcutInterface
+{
+    const PAYMENT_METHOD_VARIANT = 'paypal_express';
```

Comment:
What would be the consequences of defining the variant as `paypal` instead of `paypal_exress`?

Custom payment method generation of `adyen_paypal_express` in di.xml facades should be independent from this change. So, in theory, we can use `paypal` as the variant. Then, we can use `payment/adyen_paypal/express_show_on` path. This will will reduce the complexity of creating a different configuration path pattern in `core_config_data` table.

Therefore, button.js path can be replaced with `/paypal/button.js`.

Reply:
Creating a ticket to be handled in coming weeks. Not delaying the release